elide-datastore/elide-datastore-aggregation/src/test/java/com/yahoo/elide/datastores/aggregation/AggregationDataStoreTransactionTest.java

```diff
@@ -50,8 +54,8 @@
     // inject our own query instead of using buildQuery impl
     private class MyAggregationDataStoreTransaction extends AggregationDataStoreTransaction {
 
-        public MyAggregationDataStoreTransaction(QueryEngine queryEngine, Cache cache) {
-            super(queryEngine, cache);
+        public MyAggregationDataStoreTransaction(QueryEngine queryEngine, Cache cache, QueryLogger queryLogger) {
+            super(queryEngine, cache, queryLogger);
```

Comment:
I feel like we should have one test for query logging failures (where an exception is thrown).  Also, we should have one test for cancelation.